src/lua/trigger.c

```diff
@@ -200,7 +200,7 @@ lbox_trigger_reset(struct lua_State *L, int top, struct rlist *list,
 			trg->base.run = lbox_trigger_run;
 			trg->base.data = NULL;
 			trg->base.destroy = lbox_trigger_destroy;
-			trg->base.async = false;
+			trg->base.async = true;
```

Comment:
As I understood a trigger ends with non-zero status in case of error. If you just return -1 from lua trigger - it will not prevent other triggers to run. I didn't even found in our manual that the execution of triggers stops if case of one's fail.
The only change that this commit does is that execution become async.